examples/templates/bio_sockets.h

```diff
@@ -14,15 +18,15 @@ BIO* open_nb_socket(const char* addr, const char* port) {
     BIO_set_conn_port(bio, port);
 
     /* timeout after 10 seconds */
-    int start_time = time(NULL);
+    int start_time = (int)time(NULL);
```

Comment:
Time does not return a signed type on windows, this caused warnings.

Reply:
It's okay if the bio example doesn't get built on windows.